lib/media/track/mediatrack.js

```diff
@@ -118,24 +119,29 @@ class MediaTrack extends Track {
    * @private
    */
   _initialize() {
-    const self = this;
+    try {
```

Comment:
Do we still need to keep this try catch block?

Reply:
I think so. This fix ensures that we clean up the dummy element. but customers would still see the problem if they manage to create 75+ elements (possible on large rooms). In such cases its good to log the error for further confirmation and of the issue

Reply:
Should we throw a TwilioError instead? That way, it follows our convention when an error happens.